extension/test/unit/config/config.spec.js

```diff
@@ -250,4 +250,67 @@ describe('::config::', () => {
       )
     }
   })
+
+  it(
+    'when extra paymentMethodsToNames config is not provided, ' +
```

Comment:
can you also consider the case where a new paymentMethod is introduced, which is not provided nor defined in the default config?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/699/commits/91586bd93100923b362fcb3d28b64779038d2ad8

Reply:
Yes, this is also tested, not in this file, but here: https://github.com/commercetools/commercetools-adyen-integration/pull/699/files#diff-6df739f625bc87fcfa7a1911e1dde3516dac9d851aac40007c917607320efa31R340